bentoml/adapters/file_input.py

```diff
@@ -173,9 +174,51 @@ def from_aws_lambda_event(self, event: AwsLambdaEvent) -> InferenceTask[FileLike
         f = FileLike(bytes_=base64.decodebytes(event.get('body', "")))
         return InferenceTask(aws_lambda_event=event, data=f)
 
-    def from_cli(self, cli_args: Tuple[str]) -> Iterator[InferenceTask[FileLike]]:
-        for f in parse_cli_input(cli_args):
-            yield InferenceTask(cli_args=cli_args, data=f)
+    def from_cli(self, cli_args: Tuple[str]) -> Iterator[InferenceTask[str]]:
```

Comment:
what's the reason for changing from `Iterator[InferenceTask[FileLike]]` to `Iterator[InferenceTask[str]]`?